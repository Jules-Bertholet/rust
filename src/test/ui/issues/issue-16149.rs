extern "C" {
    static externalValue: isize;
}

fn main() {
    let boolValue = match 42 {
        externalValue => true,
        //~^ ERROR statics from `extern` blocks cannot be referenced in patterns
        _ => false,
    };
}
