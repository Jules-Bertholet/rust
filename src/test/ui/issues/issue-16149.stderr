error[E0158]: statics from `extern` blocks cannot be referenced in patterns
  --> $DIR/issue-16149.rs:7:9
   |
LL |         externalValue => true,
   |         ^^^^^^^^^^^^^

error: aborting due to previous error

For more information about this error, try `rustc --explain E0158`.
